### PR TITLE
Imakefile cleanup: Delete all ancient Vcs comments stemming from SVN …

### PR DESCRIPTION
**Pull request by sunweaver**
…times of X.org and XFree86.

 Fixes ArcticaProject/nx-libs#250.

### CODE REVIEW

**Comment by sunweaver** [comment]
@uli42: please review #254. Thanks.

File: nx-X11/config/cf/X11.tmpl [hunk 1]
```diff
@@ -1,12 +1,6 @@
 XCOMM ----------------------------------------------------------------------
 XCOMM X Window System Build Parameters and Rules
-XCOMM $XdotOrg: xc/config/cf/X11.tmpl,v 1.54 2005/12/14 23:41:52 alanc Exp $
-XCOMM $Xorg: X11.tmpl,v 1.6 2000/08/17 19:41:46 cpqbld Exp $
-XCOMM
-XCOMM
-XCOMM
-XCOMM
-XCOMM $XFree86: xc/config/cf/X11.tmpl,v 1.248 2004/02/16 04:07:37 dawes Exp $
+XCOMM ----------------------------------------------------------------------
```
Comment (uli42) [inline nx-X11/config/cf/X11.tmpl:3]: remove single empty XCOMMs?

**Review by uli42 — CHANGES_REQUESTED** [review]

File: nx-X11/config/cf/X11.tmpl (hunk 1 above)
Comment (sunweaver) [reply nx-X11/config/cf/X11.tmpl:3]: I am not getting your point here? What do you suggest?

Comment (uli42) [reply nx-X11/config/cf/X11.tmpl:3]: forget it, I was mislead by the github diff display. PR ist ok for me

**Review by uli42 — APPROVED** [review]

**Comment by sunweaver** [comment]
Closed by https://github.com/ArcticaProject/nx-libs/commit/4c59a3c58b873440573cd29eee0ead6f7dd78fcf

**Pull request closed without merging**